src-cljs/fluree_browser_sdk.cljs

```diff
@@ -0,0 +1,106 @@
+(ns fluree-browser-sdk
+  (:require [fluree.db.json-ld.api :as fluree]
+            [fluree.db.util.log :as log]))
+
+(enable-console-print!)
+
+;; define your app data so that it doesn't get over-written on reload
+(defonce app-state (atom {:product "FlureeDB APIs"
+                          :version "v0.17.0"}))
```

Comment:
Hmm... thought I had eliminated this w/ a macro. 🤔 Maybe just in the node code. I'll clean it up outside of this PR.